core/deployment/src/main/java/io/quarkus/deployment/util/WebJarUtil.java

```diff
@@ -104,15 +106,40 @@ public static Path copyResourcesForDevOrTest(LiveReloadBuildItem liveReloadBuild
             String rootFolderInJar)
             throws IOException {
         return copyResourcesForDevOrTest(liveReloadBuildItem, curateOutcomeBuildItem, launchMode, resourcesArtifact,
-                rootFolderInJar, true);
+                rootFolderInJar, true, false);
+    }
+
+    /**
+     *
+     * @param liveReloadBuildItem
+     * @param curateOutcomeBuildItem
+     * @param launchMode
+     * @param resourcesArtifact
+     * @param rootFolderInJar
+     * @param useDefaultQuarkusBranding
+     * @return
+     * @throws IOException
```

Comment:
Maybe remove all this noise? I'll test this with my GitHub App thingy.

Reply:
done